how-to-pitch/readme.md

```diff
@@ -0,0 +1,49 @@
+
+### What to tell in your story
+
+* **Grab their attention**
+You only have about 20 seconds to grab someone's attention, so start of with something, a statement for example, that grabs the interest of the receiver. Knowing who you are pitching to can greatly help you formulate something that is particular relevant for you audience.
+* **Create a context for your pitch**
+* **Introduce the problem you are trying to solve**
+* **Introduce your solution**
+* **Explain why this solution sets itself apart from others**
+* **Explain how the solution you offer is sustainable**
+* **Explain what makes you capable to offer this solution**
+* **Validate your solution** Give examples of others that have been successful in a comparable way, illustrating that your suggestion works and illustrating how your solution is different than theirs.
+* **Summarize the problem and the sollution you offer**
+
+### How to tell your story
+While telling your story there are a few key points you need to follow:
+
+* **There needs to be a natural transition** When the listeners needs think back to what you said before they can't listen to what you are saying.
+* **It needs to be understandable** Never use terms or principles without explaining them first, if there is a chance the listener won't  understand them.
+
+### Instilling trust in what you say and how you say it
+
+People buy who you are, not what you offer. Super cliche, but super true.  
+
+You can instill trust with your pitch if you try to use the follwing points while pitcing. Dont force it and dont take it to heavy.
+
+* **Be integer** Are you really trying to achieve these goals or do you have alternative motives?" Easiest one: be honest. Don't try to be more than you are and most of all: don't be an ass.
+* **Be passionate:** Are you enthusiastic enough to go beyond the obvious?
+* **Be Experienced:** Are you capable of achieving your set goals? This can be as simple as dropping a line that suggests that you did this before.
+* **Be Knowledgable:** Do you understand what you are talking about? This can be done by clearly and simply explaining complicated factors of your pitch.
+* **Be Skilled:** Are you capable of achieving the promise, or can you hire the right people?
+* **Show Leadership:** Are you capable of creating and maintaining a team? This is mostly done with the way you talk and your posture. Can you demand and maintain their attention.
+* **Show Commitment:** Are you reliable to continue even if things are not going as pleased? This sounds like simply showing enthusiasm, but this is more about making clear that you have no trouble going for the long haul.
+* **Show Vision:** Are you capable to look beyond the obvious? If you solution/pitch is original enough you got this covered.
+* **Be Realistic:** Are you realistic or are you just spewing numbers to convince me?
+* **Be Coachable:** Will you take advice from your investor? Do you have the ability to listen?
```

Comment:
would be nice to have an ending to your amazing story!
 Add a line note
View changes